ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Protocol/Crypto.hs

```diff
@@ -51,8 +56,19 @@ data TPraosStandardCrypto
 
 instance Crypto TPraosStandardCrypto where
   type DSIGN TPraosStandardCrypto = Ed25519DSIGN
-  type KES   TPraosStandardCrypto = SimpleKES Ed25519DSIGN 14 -- TODO: replace with the Sum6KES or Sum7KES
+  type KES   TPraosStandardCrypto = Sum7KES Ed25519DSIGN Blake2b_256
   type VRF   TPraosStandardCrypto = SimpleVRF
   type HASH  TPraosStandardCrypto = Blake2b_256
 
+  -- TODO This is somewhat annoying, since we want it to be based upon a runtime
+  -- value.
```

Comment:
Hmm, is there a ticket for this in `cardano-ledger-specs`?

Reply:
No ticket yet, just a branch with me playing around and a conversation on slack... I'll make one